### ISSUE TITLE
Set Electron __directory to that of the cwd of Julia

### ISSUE THREAD

**Issue by wolfiex** [JuliaGizmos/Blink.jl#263]
Currently, the root localhost directory is that of the electron install within the Blink.jl package. It would be useful to have this as the current working directory from which Julia is initiated as this would greatly aid in the reading of unorthodox binary files which may be needed within the code.

**Comment by ZHIWEN-PING** [comment]
There is a hack around this: 
1. create an HTML file
2. loadurl(w, """file:///$(abspath("test.html"))""")

**Comment by ludvax** [comment]
Agree, the root localhost directory is that of the electron install within the Blink.jl package.
I have the same issue when I use load! or loadjs! or loadcss!
In case of load! it's should be use pwd() or something like that ?
